apps/workplace/src/tests/book/new-desk-flow/new-desk-flow-success.component.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { MatDialog } from '@angular/material/dialog';
+import { createRoutingFactory, Spectator } from '@ngneat/spectator/jest';
+import { NewDeskFlowSuccessComponent } from 'apps/workplace/src/app/book/new-desk-flow/new-desk-flow-success.component';
+
+describe('NewDeskFlowSuccessComponent', () => {
+    let spectator: Spectator<NewDeskFlowSuccessComponent>;
+    const createComponent = createRoutingFactory({
+        component: NewDeskFlowSuccessComponent,
+        providers: [{ provide: MatDialog, useValue: { open: jest.fn() } }],
+        declarations: [],
+    });
+
+    beforeEach(() => (spectator = createComponent()));
+
+    it('should create component', () =>
+        expect(spectator.component).toBeTruthy());
```

Comment:
Is there any behaviour in this component that needs testing?

Reply:
Add some test for success flow view